homeassistant/components/motioneye/camera.py

```diff
@@ -144,7 +142,7 @@ def camera_add(camera: dict[str, Any]) -> None:
 class MotionEyeMjpegCamera(MotionEyeEntity, MjpegCamera):
     """motionEye mjpeg camera."""
 
-    _name: str
+    _attr_name = None
```

Comment:
Maybe this is misleading, because when calling `MjpegCamera.__init__`, `attr_name` is unconditionally set.
Let's remove it from here, and explicitly pass `name=None` to `MjpegCamera.__init__`.
```suggestion
```